core/templates/pages/about-page/about-page.component.ts

```diff
@@ -19,101 +19,76 @@
-    const results = AboutPageConstants.CREDITS_CONSTANTS.filter(
-      (credit) => credit.startsWith(startLetter)).sort();
-    return results;
-  }
-
-  onTabClick(tabName: string): Window {
-    this.windowRef.nativeWindow.location.hash = '#' + tabName;
-    this.activeTabName = tabName;
-    return this.windowRef.nativeWindow;
+    private windowRef: WindowRef,
+    private siteAnalyticsService: SiteAnalyticsService,
+    private userService: UserService,
+    private loaderService: LoaderService) {
   }
 
   getStaticImageUrl(imagePath: string): string {
     return this.urlInterpolationService.getStaticImageUrl(imagePath);
   }
 
-  ngOnInit(): void {
-    this.activeTabName = this.TAB_ID_ABOUT;
-    this.allCredits = [];
-    const letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
-    for (const letter of letters) {
-      const names = this.getCredits(letter);
-      if (names.length > 0) {
-        this.allCredits.push({letter, names});
-      }
-    }
-    const hash = this.windowRef.nativeWindow.location.hash.slice(1);
-    if (hash === 'license') {
-      this.activeTabName = this.TAB_ID_FOUNDATION;
-    } else if (this.ALLOWED_TABS.includes(hash)) {
-      this.activeTabName = hash;
-    }
+  onClickVisitClassroomButton(): void {
+    this.classroomUrl = this.urlInterpolationService.interpolateUrl(
+      '/learn/<classroomUrlFragment>', {
+        classroomUrlFragment: constants.DEFAULT_CLASSROOM_URL_FRAGMENT
+      });
+    this.loaderService.showLoadingScreen('Loading');
+    this.userService.getUserInfoAsync().then((userInfo) => {
+      this.userIsLoggedIn = userInfo.isLoggedIn();
+      this.loaderService.hideLoadingScreen();
+    });
+    this.siteAnalyticsService.registerClickVisitClassroomButtonEvent();
+    this.windowRef.nativeWindow.location.href = this.classroomUrl;
+  }
 
-    this.listOfNames = this.listOfNamesToThank
-      .slice(0, this.listOfNamesToThank.length - 1).join(', ') +
-      ' & ' + this.listOfNamesToThank[this.listOfNamesToThank.length - 1];
-    this.aboutPageMascotImgUrl = this.urlInterpolationService
-      .getStaticImageUrl('/general/about_page_mascot.webp');
+  onClickBrowseLibraryButton(): void {
+    this.siteAnalyticsService.registerClickBrowseLibraryButtonEvent();
+    this.windowRef.nativeWindow.location.href = '/community-library';
+  }
 
-    this.windowRef.nativeWindow.onhashchange = () => {
-      const hashChange = this.windowRef.nativeWindow.location.hash.slice(1);
-      if (hashChange === 'license') {
-        this.activeTabName = this.TAB_ID_FOUNDATION;
-        this.windowRef.nativeWindow.location.reload(true);
-      } else if (this.ALLOWED_TABS.includes(hashChange)) {
-        this.activeTabName = hashChange;
-      }
-    };
+  onClickCreateLessonButton(): void {
+    this.siteAnalyticsService.registerCreateLessonButtonEvent();
+    this.windowRef.nativeWindow.location.href = (
+      '/creator-dashboard?mode=create');
   }
+
+  ngOnInit(): void {}
```

Comment:
Why we need this if we are not doing anything during init?

Reply:
removed